compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/PsiUtils.kt

```diff
@@ -91,62 +92,85 @@ public fun KtAnnotated.requireAnnotation(
 }
 
 @ExperimentalAnvilApi
+public fun KtAnnotated.requireAnnotations(
+  fqName: FqName,
+  module: ModuleDescriptor
+): List<KtAnnotationEntry> {
+  return findAnnotations(fqName, module)
+    .ifEmpty { null }
```

Comment:
takeIfNotEmpty()? You just created the extension.

Reply:
For that I had to change the signature slightly. It's working now.
```kotlin
public inline fun <C, T> C.takeIfNotEmpty(): C? where C : Collection<T> = ifEmpty { null }
```